wireless_modules/middle_module/README.md

```diff
@@ -0,0 +1,62 @@
+# DAS - MIDDLE WIRELESS MODULE
+
+[![All Contributors](https://img.shields.io/badge/all_contributors-6-orange.svg?style=flat-square)](#contributors)
```

Comment:
We only need the all contributors stuff in the root readme, can remove from here (the stuff at the bottom of the file too)